docs/tutorial/notifications.md

```diff
@@ -123,39 +137,46 @@ main process or the renderer process), use the userland module
 which uses native Node addons to send `ToastNotification` and `TileNotification` objects.
 
 While notifications including buttons work with `electron-windows-notifications`,
-handling replies requires the use of [`electron-windows-interactive-notifications`](https://github.com/felixrieseberg/electron-windows-interactive-notifications), which
-helps with registering the required COM components and calling your Electron app with
-the entered user data.
+handling replies requires the use of
+[`electron-windows-interactive-notifications`](https://github.com/felixrieseberg/electron-windows-interactive-notifications),
+which helps with registering the required COM components and calling your
+Electron app with the entered user data.
 
 #### Quiet Hours / Presentation Mode
 
-To detect whether or not you're allowed to send a notification, use the userland module
-[electron-notification-state](https://github.com/felixrieseberg/electron-notification-state).
+To detect whether or not you're allowed to send a notification, use the
```

Comment:
I don't think `userland` should be in code blocks here. Maybe replace with "third-party" to avoid confusion with `electron-userland`?